tools/cli/app.go

```diff
@@ -27,7 +27,7 @@ import (
 const (
 	// Version is the controlled version string. It should be updated every time
 	// before we release a new version.
-	Version = "0.8.1"
+	Version = "0.8.2"
```

Comment:
I think we have 0.8.3.

Reply:
Only for CLI. Compatible with server 0.8.*